libcaf_test/caf/test/reporter.cpp

```diff
@@ -170,7 +199,10 @@ class default_reporter : public reporter {
   }
 
   auto colored() {
-    return colorizing_iterator{colorizing_iterator::normal, &std::cout};
+    if (plain_output_)
+      return colorizing_iterator{colorizing_iterator::off, &std::cout};
+    else
+      return colorizing_iterator{colorizing_iterator::normal, &std::cout};
```

Comment:
The if-else really only translates the boolean to the state flag, so putting this upfront makes the control flow a bit more clear IMHO:

```suggestion
    auto state = plain_output_ ? colorizing_iterator::off : colorizing_iterator::normal;
    return colorizing_iterator{state, &std::cout};
```